profiles/visual-studio-17

```diff
@@ -1,4 +1,4 @@
-include(msvc-common)
+include(visual-studio-common)
 [settings]
 os=Windows
 os_build=Windows
```

Comment:
`os_build` should be an error in Conan 2

Reply:
Since I'm no longer using the visual studio compiler for CCCI, I'm removing the changes I had made.